app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1020,4 +1020,9 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = false,
         featureFlag = FeatureFlags.addressesFeature
     )
+
+    var openNextTabInDesktopMode by booleanPreference(
```

Comment:
Our list of settings is getting a bit long. Would you mind adding a comment to describe this preference for future documentation purposes?